spec/models/grant_creator_request_spec.rb

```diff
@@ -1,5 +1,93 @@
 require 'rails_helper'
 
 RSpec.describe GrantCreatorRequest, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it { is_expected.to respond_to (:requester) }
+  it { is_expected.to respond_to (:request_comment) }
+  it { is_expected.to respond_to (:reviewer) }
+  it { is_expected.to respond_to (:status) }
+
+  describe '#validations' do
+    let(:request)          { build(:grant_creator_request) }
+
+    it 'validates a valid request' do
+      expect(request).to be_valid
+    end
+
+    it 'requires a request_comment' do
+      request.request_comment = nil
+      expect(request).not_to be_valid
+      expect(request.errors).to include(:request_comment)
+    end
+
+    it 'disallows request from system_admin' do
+      request.requester.update_attribute(:system_admin, true)
+      expect(request).not_to be_valid
+      expect(request.errors[:base]).to include I18n.t('activerecord.errors.models.grant_creator_request.attributes.base.is_system_admin')
+    end
+
+    it 'disallows request from grant_creator' do
+      request.requester.update_attribute(:grant_creator, true)
+      expect(request).not_to be_valid
+      expect(request.errors[:base]).to include I18n.t('activerecord.errors.models.grant_creator_request.attributes.base.has_grant_creator_access')
+    end
+
+    it 'disallows request from user with an exisiting pending request' do
+      request.save
+      expect(FactoryBot.build(:grant_creator_request, requester: request.requester)).not_to be_valid
+    end
+
+    it 'allows request from user with an exisiting rejected request' do
+      request.update_attribute(:status, 'Approved')
```

Comment:
not sure if the description matches this test

Reply:
it did not